modules/new_serverpod_auth/serverpod_auth_idp/serverpod_auth_idp_server/lib/src/providers/email/business/email_account_server_exceptions.dart

```
@@ -0,0 +1,68 @@
+/// Base exception for all email related errors.
+///
+/// These exceptions are for internal purposes only and must not be exposed to
+/// clients. For such, see the adequate serializable exceptions.
+sealed class EmailServerException implements Exception {}
```

Suggestion: Create a sub sealed group that extends `EmailServerException` for the exceptions that could be thrown in different scenarios.

This makes it possible to catch all exceptions from a scenario without capturing other cases.